src/identity/AttesterIdentity.ts

```diff
@@ -313,7 +313,7 @@ export default class AttesterIdentity extends Identity {
     const bc = await getCached()
     const tx = bc.portablegabi.buildUpdateAccumulatorTX(acc)
     this.accumulator = acc
-    await bc.portablegabi.signAndSend(tx, this.signKeyringPair)
+    await bc.submitTx(this, tx)
```

Comment:
Why is this change necessary?

Reply:
Signing and sending Tx with gabi chain can result in nonce collision as well.

Reply:
Different instances of our Chain will use the `SubmittableExtrinsic` specific methods to sign and or send the tx, and is therefore completely independent from the chain instance, after the tx's creation/instantiation.